README.md

```diff
@@ -15,6 +15,17 @@ super-fast querying and searching, as well as serves the merkle proofs needed to
 The API specification is located here https://github.com/metaplex-foundation/api-specifications
 This spec is what providers of this api must implement against.
 
+#### GRPC-INGEST [/grpc-ingest/README.md](/grpc-ingest/README.md)
+
+This repo also houses the GRPC-INGEST component. This is a reimplementation of nft-ingester
+This components separates grpc and ingester into two separate components. This is to allow for more flexibility in the future also increases the performance of the system.
```

Comment:
```suggestion
The grc ingester allows the DAS index stack to receive solana change events from a Yellowstone GRPC endpoint. All the largest RPC providers offer this API so you are able to use a shared event stream instead of running a dedicated validator like you have to with the plerkle geyser plugin.
```